infra/terraform/test_fixtures/terraform-google-iam.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+  // Roles required by CI service accounts to run the IAM
+  // integration tests.
+  iam_required_roles = [
+    "roles/resourcemanager.organizationAdmin",
+    "roles/compute.xpnAdmin",
```

Comment:
Why is `roles/compute.xpnAdmin` included? The module does not appear to make use of shared VPCs.

Reply:
Addressed in d562056 Fixup - Remove roles/compute.xpnAdmin

Reply:
@aaron-lane xpnAdmn is less wide option for

> Subnetwork:
> Project compute admin: Full control of Compute Engine resources.
> Project compute network admin: Full control of Compute Engine networking resources.
> Project custom: Add compute.subnetworks.getIamPolicy	and compute.subnetworks..setIamPolicy permissions

https://github.com/terraform-google-modules/terraform-google-iam

It contains compute.subnetworks.getIamPolicy compute.subnetworks.setIamPolicy

Reply:
xpnAdmin is Shared VPC admin and needs to be granted at Org Level.  Network Admin should be used instead which contains `compute.subnetworks.*` https://cloud.google.com/iam/docs/understanding-roles#compute-engine-roles